yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1400,3 +1410,27 @@ func validateAssessmentMetadataDirFlag() {
 		}
 	}
 }
+
+func validateAndSetTargetDbVersionFlag() error {
```

Comment:
do we need to return error from this function? since we are not adding any context to the err message from the caller func

Reply:
Yeah, alternative could be to just ErrExit directly. 
I just thought it would be good to follow the pattern of returning errors everywhere; only utils.ErrExit in the cobra command's PreRun/Run functions (and ofc prompts)

Reply:
ok, yeah the pattern makes sense. at somepoint we can refactor other validates() also....